### ISSUE TITLE
Level of Beastmaster when /BST should effect Charm success rate but not Charm duration

### ISSUE THREAD

**Issue by him911** [EdenServer/community#1760]
### Checklist

- [yes] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [yes] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [yes] I have provided my client version in the details. (type /ver into your game window)

### Details
Charm duration and charm success rate are seperate entities. 
While success rate may rely on your bst lvl , duration does not
https://www.bg-wiki.com/bg/Charm_Duration#References

Right now bst as a subjob is resulting in charm lasting 20-30 seconds as opposed to giving me the full duration of a regular charm without charm+ gear. 

Charming a Decent challenge at 75xjob/37bst  should yield the standard 10 minute charm

**Comment by xizeta** [comment]
From the link you posted
```
The Base Charm Duration is modified by the level difference between the player and their target. 
```

**Comment by xizeta** [comment]
Firstly, we fixed the Charm success rate for /BST which is absolutely correct. BST main uses level difference as a base, where +CHR modifies rate and +Charm modifies duration.

It really made no sense for the subjob mechanic to be any different than this. The wiki also does not contradict this view. The point of the penalty is to prevent every job from just having free max level BST powers, which they could with Charisma gear and only lv37.

However, this is SE and they do a lot of things which don't make any sense. So I tested on retail, by leveling a BST to 49 and subbing it at 99. The lv49 does indeed determine the Charm rate, however it did not influence the duration.

Of course, we have no proof this wasn't change after era but this will be adjusted to best match the accuracy of the information we now have available.

I reworded the issue for clarity.
